modules/ztp-pre-staging-partitioning.adoc

```diff
@@ -0,0 +1,207 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp_far_edge/ztp-pre-staging-tool.adoc
+
+:_module-type: PROCEDURE
+[id="ztp-partitioning_{context}"]
+= Partitioning the disk
+
+To run the full pre-caching process, you have to boot from a live ISO and use the {factory-prestaging-tool} from a container image to partition and pre-cache all the artifacts required.
+
+[WARNING]
+====
+{op-system} requires the disk to not be in use when the disk is about to be written by an {op-system} image.
```

Comment:
I think we are missing why we are saying that. I'll say something like: "A live ISO or RHCOS live ISO is required since the disk does not have to be in use when the operating system (RHCOS) is written to the device during the provisioning. Notice that this way, single-disk servers can also be enabled with this procedure"